website/docs/r/rde_interface.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_rde_interface"
+sidebar_current: "docs-vcd-resource-rde-interface"
+description: |-
+   Provides the capability of creating, updating, and deleting Defined Interfaces in VMware Cloud Director.
+---
+
+# vcd\_rde\_interface
+
+Provides the capability of creating, updating, and deleting Defined Interfaces in VMware Cloud Director.
+
+-> Creating, updating and deleting Defined Interfaces requires System administrator privileges.
+
+Supported in provider *v3.9+*
+
+## Example Usage
+
+```hcl
+resource "vcd_rde_interface" "my_interface" {
+  vendor    = "bigcorp"
+  namespace = "tech"
+  version   = "1.2.3"
+  name      = "BigCorp Interface"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `vendor` - (Required) The vendor of the Defined Interface.
+* `namespace` - (Required) A unique namespace associated with the Defined Interface.
+* `version` - (Required) The version of the Defined Interface. Must follow [semantic versioning](https://semver.org/) syntax.
```

Comment:
We need the "3-tuple key" explanation in the main section of this doc page. Something similar to what you have in the PR description.

Reply:
Done!